src/Adapter/MSTest.TestAdapter/Discovery/AssemblyEnumerator.cs

```diff
@@ -77,9 +76,9 @@ public AssemblyEnumerator(MSTestSettings settings)
     internal ICollection<UnitTestElement> EnumerateAssembly(string assemblyFileName, out ICollection<string> warnings)
     {
         DebugEx.Assert(!StringEx.IsNullOrWhiteSpace(assemblyFileName), "Invalid assembly file name.");
-
         var warningMessages = new List<string>();
         var tests = new List<UnitTestElement>();
+        var fixturesTests = new HashSet<string>();
```

Comment:
We are not using this, can it be moved down?

Reply:
I kept the declaration here with the other lists. It is passed to every type so that we do not add duplicate fixtures for the same type.